src/lib/Commands/TestCommand/index.js

```diff
@@ -268,22 +281,30 @@ class TestCommand extends AbstractCommand {
     });
   }
 
-  /**
-   * Prepare source code
-   * @param testFile
-   * @return {{agent: string, device: string}}
-   * @private
-   */
+  // Prepare source code
+  // @param testFile
+  // @return {{agent: string, device: string}}
+  // @private
+
   _getSessionCode(testFile) {
     let agentCode, deviceCode;
 
-    /* [info] */
+    // [info]
     this._info(c.blue('Using ') + testFile.type + c.blue(' test file ') + testFile.name);
 
     // read/process test code
-    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim();
-    this._codeProcessor.variables.__FILE__ = path.basename(testFile.path);
-    testCode = this._codeProcessor.process(testCode);
+    let testCode = fs.readFileSync(testFile.path, 'utf-8').trim().replace(LINE_AND_FILE_REGEXP, "@{__");
+    var environmentVars = "";
+    for (var prop in process.env) {
+      if (prop !== BUILD_API_KEY_ENV_VAR) { //deny to access for BUILD_API_KEY_ENV_VAR
+        // Replace #{env:...} with @{...} if it is needed
+        let propertyRegExp = new RegExp('\\#\\{env:\\s*' + prop + '\\s*\\}', 'g');
```

Comment:
DONE: warning is added. Format is:
[+31.02/0.00s warning] Can't replace: #{env:PREDIX_UAA_URL},#{env:PREDIX_CLIENT_ID},#{env:PREDIX_CLIENT_SECRET},#{env:PREDIX_ASSET_URL},#{env:PREDIX_ASSET_ZONE_ID},#{env:PREDIX_TIME_SERIES_INGEST_URL},#{env:PREDIX_TIME_SERIES_ZONE_ID}